src/erlcloud_mon.erl

```diff
@@ -216,33 +217,112 @@ put_metric_data(Namespace, MetricName, Value, Unit, Timestamp) ->
          ),
     mon_simple_query(Config, "PutMetricData", Params).
 
+
+%%------------------------------------------------------------------------------
+%% @doc CloudWatch API - GetMetricStatistics - Easy average version
+%% Gets average stats at 60 second intervals for the given metric on the given instance for the given interval
+%% @end
+%%------------------------------------------------------------------------------
+-spec get_metric_statistics(
+        MetricName  ::string(),
+        StartTime   ::datetime() | string(),
+        EndTime     ::datetime() | string(),
+        InstanceId  ::string()
+       ) -> term().
+
+get_metric_statistics(
+  MetricName,
+  StartTime,
+  EndTime,
+  InstanceId) ->
+          get_metric_statistics(
+              "AWS/EC2",
+              MetricName,
+              StartTime,
+              EndTime,
+              60,
+              "",
+              ["Average","Maximum"],
+              [{'InstanceId', InstanceId}]).
```

Comment:
Is `InstanceId` supposed to be an atom or a string? Looking at the spec for `get_metric_statistics/8`, it should be a string so either this line or that spec should change.